docs/user/troubleshooting.md

```diff
@@ -0,0 +1,91 @@
+# Installer Troubleshooting
+
+Unfortunately, there will always be some cases where OpenShift fails to install properly. In these events, it is helpful to understand the likely failure modes as well as how to troubleshoot the failure.
+
+## Common Failures
+
+### No Workers Nodes Created
+
+The installer doesn't provision worker nodes directly, like it does with master nodes. Instead, the cluster relies on the Machine API Operator, which is able to scale up and down nodes on supported platforms. If more than fifteen to twenty minutes (depending on the speed of the cluster's Internet connection) have elapsed without any workers, the Machine API Operator needs to be investigated.
+
+The status of the Machine API Operator can be checked by running the following command from the machine used to install the cluster:
+
+```sh
+oc --config=${INSTALL_DIR}/auth/kubeconfig --namespace=openshift-cluster-api get pods
+```
+
+If the API is unavailable, that will need to be [investigated first](#api-is-unavailable).
+
+The previous command should yield output similar to the following:
+
+```sh
+NAME                                             READY     STATUS    RESTARTS   AGE
+clusterapi-manager-controllers-774dc4557-nx5xq   3/3       Running   0          4h
+machine-api-operator-7894d8f85-lq2ts             1/1       Running   0          4h
+```
+
+The logs for the machine-controller container within the `clusterapi-manager-controllers` pod need to be checked to determine why the workers haven't been created. That can be done with the following (the exact name of the pod will need to be substituted):
+
+```sh
+oc --config=${INSTALL_DIR}/auth/kubeconfig --namespace=openshift-cluster-api logs clusterapi-manager-controllers-774dc4557-nx5xq --container=machine-controller
+```
+
+### API is Unavailable
```

Comment:
```suggestion
### Kubernetes API is Unavailable
```
openshift and kube API are now on different pods

Reply:
A dedicated section on how to debug a broken Openshift API server would also be very helpful for BYOR team

Reply:
> A dedicated section on how to debug a broken Openshift API server...

Punt!  Lets get the existing content pollished and landed first; we can add more later.

Reply:
> openshift and kube API are now on different pods

Addressed with 8d7f165 -> cce0561.